simulation_ws/src/cloudwatch_simulation/launch/world_turtlebot_navigation.launch.py

```diff
@@ -0,0 +1,99 @@
+# *******************************************************************************/
+# This launch file serves as a template for navigation launch files.
+# It takes two key argument:
+# - world_launch_file, which specifies the world launch file name
+# - world_package, which specifies the actual package name that stores the world
+# *******************************************************************************/
+import os
+import sys
+
+import launch
+import launch_ros.actions
+from ament_index_python.packages import get_package_share_directory
+
+TURTLEBOT3_MODEL = os.environ.get('TURTLEBOT3_MODEL', 'waffle_pi')
```

Comment:
Does it make sense to default to waffle_pi when the instruction requires TURTLEBOT3_MODEL to be set, maybe its better to return an error here?

Reply:
(nitpick)

Reply:
The launch file that requires TURTLEBOT3_MODEL is [turtlebot3_navigation/launch/move_base.launch](https://github.com/ROBOTIS-GIT/turtlebot3/blob/master/turtlebot3_navigation/launch/move_base.launch#L3). I've created a [PR](https://github.com/ROBOTIS-GIT/turtlebot3/pull/675) to make it optional and will modify the README to make the environment variable optional once the PR is merged.